frontend/src/types/profile.ts

```diff
@@ -0,0 +1,19 @@
+import { ReviewStatus } from './runnerPost';
+
+export interface RunnerProfileResponse {
```

Comment:
```suggestion
export interface GetRunnerProfileResponse {
```
기존 타입들은 메서드명을 붙여줬는데, GET을 빼는 것도 괜찮아 보이긴 하네요.
어떻게 생각하나요?